python-backend/wolfram_client.py

```diff
@@ -66,16 +66,21 @@ def limit(expression: str) -> str:
             logger.error("Failed to obtain Wolfram API result", e)
 
     @staticmethod
-    def closed_form(expression: str) -> str:
+    def closed_form(expression: str) -> dict[str, str]:
         """
         Query the Wolfram results API for limits of a string expression
         :param expression: Mathematical expression for which we would like to compute the limit(s)
         :return: Wolfram API response "subpods" with the computed limit(s)
         """
         try:
             result = WolframClient.ask(query=expression, include_pod="PossibleClosedForm")
+            print(result)
             # only want to return: queryresult -> pods[0] -> subpods
-            return result["queryresult"]["pods"][0]["subpods"]
+            # infos has metadata on the subpods e.g. the constant names and links to reference content
+            # Note: the index of the infos does not line up with the index of the closed form results
+            subpods = result["queryresult"]["pods"][0]["subpods"]
+            meta = result["queryresult"]["pods"][0]["infos"] if "infos" in result["queryresult"]["pods"][0] else []
```

Comment:
Nit: Consider `meta = result["queryresult"]["pods"][0].get("infos", [])` to avoid the double lookup.

Reply:
Always forget that syntax when I don't use it for a while